app/templates/add.html

```diff
@@ -31,7 +31,6 @@
       <div id="textcolorcorrector" class="row-fluid">
         <div class="text-center">Please Select the Room You Wish To Reserve</div>
           <br>
-          <div class="text-center">Warning: After you select the room, you have 2 minutes to complete your reservation.</div>
```

Comment:
This file doesn't exist anymore since we have gotten rid of ALL the front-end. You need to sync your branch with Master.